src/cursor.js

```diff
@@ -92,8 +92,21 @@ export default class Cursor {
     // Without setting focus() Firefox is not happy (seems setting a selection is not enough.
     // Probably because Firefox can handle multiple selections).
     if (this.win.document.activeElement !== this.host) {
+      const previousWindowTop = $(window).scrollTop()
```

Comment:
You must always use the `window` of the cursor and not the global `window`. Editable is set up to work with multiple iframes.

Reply:
Good to know.